lms/templates/static_htmlbook.html

```diff
@@ -118,7 +118,7 @@
 <%include file="/courseware/course_navigation.html" args="active_page='htmltextbook/{0}'.format(book_index)" />
 
     <div id="outerContainer">
-        <main id="main" aria-label="Content" tabindex="-1">
+        <main id="main" aria-label="${_('Content')}" tabindex="-1">
```

Comment:
Should probably be a separate PR as this impacts MANY files. I've made [a ticket](https://openedx.atlassian.net/browse/AC-509) to follow this work.